packages/server/lib/util/chromium_flags.ts

```diff
@@ -36,7 +36,7 @@ const DEFAULT_FLAGS = [
   'enable-automation',
   'disable-print-preview',
   'disable-component-extensions-with-background-pages',
-
+  'disable-infobars',
```

Comment:
I thought this flag was removed? https://chromium-review.googlesource.com/c/chromium/src/+/1599303

Reply:
Cool.... https://source.chromium.org/chromium/chromium/src/+/main:third_party/devtools-frontend/src/node_modules/chromium-bidi/lib/esm/bidiServer/BrowserInstance.js;l=50?q=disable-infobars&sq=&ss=chromium

Reply:
Yeah, it seems like it was added back since it definitely controls whether or not some of the toolbars are displayed.